lookout/style/format/tests/test_analyzer_integration.py

```diff
@@ -97,23 +95,24 @@ class AnalyzerIntegrationTests(BaseAnalyzerIntegrationTests):
     @classmethod
     def setUpClass(cls):
         parent = Path(__file__).parent.resolve()
-        cls.base_dir = str(parent)
-        cls.jquery_dir = str(parent / "jquery")
+        cls.base_dir_ = tempfile.TemporaryDirectory(dir=str(parent))
+        cls.base_dir = cls.base_dir_.name
+        cls.jquery_dir = os.path.join(cls.base_dir, "jquery")
         # str() is needed for Python 3.5
         with tarfile.open(str(parent / "jquery.tar.xz")) as tar:
             tar.extractall(path=cls.base_dir)
 
     @classmethod
     def tearDownClass(cls):
-        shutil.rmtree(cls.jquery_dir)
+        cls.base_dir_.cleanup()
 
     def test_review(self):
         launch_server(
             "review",
-            "fbd32214d1b08e09278f39c77979658cabde6c4d",
-            "931442679ad04ebadad2b70a9f938fb0bc64d537",
+            "HEAD" + "^" * 9,
```

Comment:
Do we really need to use HEAD instead of hashes? I am very suspicious to the stability

Reply:
There are multiple places where you use HEAD8^ and HEAD9^, so would be nice to declare them as global constants